examples/raw.rs

```diff
@@ -21,7 +21,7 @@ fn print() {
     let mut cnt = 0;
     backtrace::trace(|frame| {
         let ip = frame.ip();
-        print!("frame #{:<2} - {:#02$x}", cnt, ip as usize, HEX_WIDTH);
+        print!("frame #{cnt:<2} - {:#0HEX_WIDTH$x}", ip as usize);
```

Comment:
This introduces active confusion into reading this print statement because these are now "out of order", i.e. I am no longer reading all the format arguments strictly "left to right".